addons/website_sale/models/res_config_settings.py

```diff
@@ -67,6 +67,20 @@ class ResConfigSettings(models.TransientModel):
     confirmation_email_template_id = fields.Many2one(
         related='website_id.confirmation_email_template_id', readonly=False
     )
+    send_rating_emails = fields.Boolean(
```

Comment:
Nit, for consistency:
```suggestion
    send_order_rating_emails = fields.Boolean(
```